exercises/09-documentation.md

```diff
@@ -63,11 +63,11 @@ slows down the test.
 
 One final touch we should add to our test suite is to update the test tags.
 If this was a test with a real requirement linked to a requirements management
-tool, we should have the ticket id as a tag already. Well, it's not, so we can
-ignore that. However, we added a `wip` at the beginning of the training for our
-test suite. Now our test is finished, so we can safely take that away.
+tool, we should have the ticket id as a tag already. In this workshop, it's not, so we
```

Comment:
I believe the Academy terminology uses "worksop" for other things, whereas this is a "training"
```suggestion
tool, we should have the ticket id as a tag already. In this training, it's not, so we
```

Reply:
Yes, the correct term is `training`. :)